### DISCUSSION TITLE
tutorials/voyage-part1/03-agreeing-on-a-schema

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#154] [course feedback]
# tutorials/voyage-part1/03-agreeing-on-a-schema

Use a schema-first approach to break down a user interface and design the schema for your application

https://www.apollographql.com/tutorials/voyage-part1/03-agreeing-on-a-schema

**Comment by gajjardarshithasmukhbhai** [comment]
Very Informative One, kudos to Apollo Team👏

**Reply by stephan-buckmaster** [reply]
I didn't understand why the `LocationReviewInput` type has 

    locationId: String! # Reviews subgraph

I expected this:

    locationId: Id! # Reviews subgraph

**Comment by otlyn** [comment]
Great tutorials. Thanks!
